pytorch_lightning/accelerators/tpu_backend.py

```diff
@@ -316,3 +318,6 @@ def transfer_distrib_spawn_state_on_fit_end(self, model, mp_queue, results):
                 last_path = re.sub('.ckpt', '.tmp_end.ckpt', best_model_path)
                 atomic_save(model.state_dict(), last_path)
             mp_queue.put(last_path)
+
+    def broadcast(self, obj, src=0):
```

Comment:
please add it to the tpu backend and not the dist library

Reply:
cool. Will move it there 👍